experimental/origin-dapp2/src/pages/listing/ListingDetail.js

```diff
@@ -16,7 +17,13 @@ class ListingDetail extends Component {
 
     return (
       <div className="listing-detail">
-        <div className="category">{listing.categoryStr}</div>
+        <div className="header">
```

Comment:
I'd prefer to abstract this `header` wrapper with the category and badge into its own component (instead of `_ListingBadge`). Is it something you'd be OK with, or are there any dragons I'm not aware of?

Reply:
Hmm this looks fine as is to me... unless we're going to be using the component in more than 2 places I'm not sure it's worth abstracting?

Reply:
Makes sense, won't touch it further. 😊